syntax = "proto3";

package grpc;

service DianPing{
  rpc Recommend(RecommendRequest) returns (RecommendReply);
}

message RecommendRequest{
  int32 user_id = 1;
}
message RecommendReply{
  repeated int32 shop_ids = 1;
}